1MI0800224/week2/week2_task1.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+double ReadFromFile(ifstream& file){
+    if(!file.is_open())
+        return -1;
+    double sum, multiplication;
+    file>>sum; file>>multiplication;
+    return multiplication-sum;
+}
+
+int main(){
+    double x, y, z;
+    cin>>x>>y>>z;

```

Comment:
If I am to be running this program - I have to have a :crystal_ball: in order to guess what's expected from me.. since no user prompts have been provided :crying_cat_face: